src/main/resources/skillresponse/response.properties

```diff
@@ -116,13 +119,16 @@ askForCityAfterStop.personalized={0}, You haven''t set your region yet. In what
 duplicatedStops.personalized=We found {0} stops associated with the stop number. Did you mean the {1} stop?
 #{0} stopName
 duplicatedStopConfirm.personalized=Ok, what about the {0} stop?
+#{1} cityName, {2} stopNumber
```

Comment:
I can't comment on line 97 directly, but let's change:
arrivalInfoFormat.personalized=Hi {0}, {1}`

...to:
`arrivalInfoFormat.personalized=Hi {0} {1}`

It's a bit annoying to wait for the pause after your name for arrival times every time you invoke the skill.

Reply:
Done